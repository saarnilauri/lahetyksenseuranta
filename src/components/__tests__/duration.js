import React from "react";
import renderer from "react-test-renderer";

import Duration from "../duration";

describe("Duration", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Duration className="test" seconds={3600}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
