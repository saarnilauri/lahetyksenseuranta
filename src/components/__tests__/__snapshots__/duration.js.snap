// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Duration renders correctly 1`] = `
<time
  className="test"
  dateTime="P3600S"
>
  1:00:00
</time>
`;
